testsuite/integration/web/pom.xml

```diff
@@ -687,5 +687,487 @@
+                                        <jboss-fork-embedded>${galleon.fork.embedded}</jboss-fork-embedded>
+                                    </plugin-options>
+                                    <feature-packs>
+                                        <feature-pack>
+                                            <groupId>${testsuite.ee.galleon.pack.groupId}</groupId>
+                                            <artifactId>${testsuite.ee.galleon.pack.artifactId}</artifactId>
+                                            <version>${testsuite.ee.galleon.pack.version}</version>
+                                            <inherit-configs>false</inherit-configs>
+                                            <inherit-packages>false</inherit-packages>
+                                        </feature-pack>
+                                    </feature-packs>
+                                    <layers>
+                                        <layer>cloud-server</layer>
+                                    </layers>
+                                    <extra-server-content-dirs>
+                                        <extra-content>${basedir}/target/wildfly-users</extra-content>
+                                        <extra-content>${basedir}/target/external-taglib-module</extra-content>
+                                    </extra-server-content-dirs>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+
+
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <executions>
+                            <!-- Disable default-test -->
+                            <execution>
+                                <id>default-test</id>
+                                <phase>none</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                            </execution>
+
+                            <!-- Tests against the datasource-web-server without legacy security -->
+                            <execution>
+                                <id>bootable-jar-web-profile-layers.surefire</id>
+                                <phase>test</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration>
+                                    <environmentVariables>
+                                        <JBOSS_HOME>${jboss.dist}</JBOSS_HOME>
+                                    </environmentVariables>
+                                    <systemPropertyVariables>
+                                        <bootable.jar>${project.build.directory}/test-wildfly-web-profile.jar</bootable.jar>
```

Comment:
Is the `install.dir` property left to be assigned to the default value (i.e. temp dir) wanted? I am just checking here, since I've seen it used quite often, maybe for testing/debugging purposes. WDYT?

Reply:
I can for sure set it to any value desired, though I'm not sure what would the ideal value should be...

Reply:
It depends. What I noticed from the modules that I've done and reviewed already, when packaging just one server, then the `${project.build.directory}/wildfly` expression is used. In your case it could be instead done as in here: https://github.com/fabiobrz/wildfly/pull/1/files#diff-4b8d9b59b6ff474b31a2557a6ee42c72R708